### PR TITLE
Add Japanese translation

### PR DESCRIPTION
**Pull request by Dirain1700**
Ref: #8608

### CODE REVIEW

File: translations/japanese/core-commands.ts [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+		"Setting status messages in /busy is no longer supported. Set a status using /status.": "/busyでのステータスメッセージの設定はサポートされなくなりました。ステータスメッセージの設定は、/statusを使って行ってください。",
+		"Setting status messages in /away is no longer supported. Set a status using /status.": "/awayでのステータスメッセージの設定はサポートされなくなりました。ステータスメッセージの設定は、/statusを使って行ってください。",
+		"User '${target}' not found.": "ユーザー名'${target}'は見つかりませんでした。",
+		"${targetUser.name} does not have a status set.": "${targetUser.name}はステータスメッセージを設定していません。",
+		"${targetUser.name}'s status \"${targetUser.userMessage}\" was cleared by ${user.name}${displayReason}": "${targetUser.name}のステータスメッセージは${user.name}によって消去されました。${displayReason}",
+		"You don't have a status message set.": "ステータスメッセージを設定していません。",
+		"You have cleared your status message.": "ステータスメッセージを消去しました。",
+		"This user has not played any ladder games yet.": "このユーザーはladderが有効なバトルをしていません。",
+		"W[TN: initial for Wins]": "'Wins'の頭文字",
+		"L[TN: initial for Losses]": "'Losses'の頭文字",
```
Comment (AnnikaCodes) [inline translations/japanese/core-commands.ts:29]: These are for columns with just a number in them; if it's possible to make these abbreviations shorter that would probably look better. If not then that's okay too; nothing will break, but the columns will be a bit wide.

Comment (Dirain1700) [reply translations/japanese/core-commands.ts:29]: oh, I missed. Thank you!

**Comment by Dirain1700** [comment]
Can you re-run workflow?

**Comment by AnnikaCodes** [comment]
(You'll need to push a commit or run `git push --force` to force GitHub to rebase with the latest commits while running the workflow)

**Comment by AnnikaCodes** [comment]
(Please resolve merge conflicts)

File: translations/japanese/core-commands.ts [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+		"${targetUser.name} was kicked from a battle by ${user.name} ${displayReason}": "${targetUser.name}は${user.name}によってバトルからキックされました。(${displayReason})",
+		"You can only set the timer from inside a battle room.": "タイマーはバトル部屋の中で設定できます。",
+		"This game's timer is managed by a different command.": "このゲームのタイマーは他のコマンドで管理されています。",
+		"The game timer is OFF.": "ゲームのタイマーがオフになりました。",
+		"The game timer is ON (requested by ${requester})": "ゲームのタイマーがオンになりました(${requester}が設定しました)",
+		"Access denied.": "アクセスが拒否されました。",
+		"Timer was turned off by staff. Please do not turn it back on until our staff say it's okay.": "スタッフがタイマーをオフに設定しました。スタッフの許可を得るまで、元に戻さないでください。",
+		"The timer is already off.": "タイマーは既にオフに設定されています。",
+		"\"${target}\" is not a recognized timer state.": "\"${target}\"は認識できないタイマーの状態です。",
+		"Forcetimer is now OFF: The timer is now opt-in. (set by ${user.name})": "",
```
Comment (AnnikaCodes) [inline translations/japanese/core-commands.ts:107]: Why not translate this one?

**Review by AnnikaCodes — APPROVED** [review]
I don't speak Japanese, but this looks fine to me in terms of formatting. Thanks!

File: translations/japanese/core-commands.ts (hunk 2 above)
Comment (Dirain1700) [reply translations/japanese/core-commands.ts:107]: oops I forgot it… Thank you!

File: translations/japanese/core-commands.ts [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+		"The replay for this battle is already set to hidden.": "このバトルのリプレイはすでに隠されています。",
+		"${user.name} hid the replay of this battle.": "${user.name}がこのバトルのリプレイを非表示にしました。",
+		"You can only do this in battle rooms.": "これはバトル部屋のみで使えます。",
+		"You can only add a Player to unrated battles.": "プレイヤーを追加できるのはレートが有効でないバトルのみです。",
+		"Player must be set to \"p1\" or \"p2\", not \"${target}\".": "プレイヤーは\"p1\"か\"p2\"に設定しなければなりません。\"${target}\"ではありません。",
+		"User ${name} not found.": "ユーザー ${name} は見つかりませんでした。",
+		"User ${name} must be in the battle room already.": "ユーザー\"${name}\"はすでにバトル部屋に入室しています。",
+		"This room already has a player in slot ${target}.": "スロット${name}にはすでにユーザーがいます。",
+		"${targetUser.name} is already a player in this battle.": "\"${targetUser.name}\"はすでにプレイヤーになっています。",
+		"${name} was added to the battle as Player ${playerNum} by ${user.name}.": "",
```
Comment (Dirain1700) [inline translations/japanese/core-commands.ts:91]:
```suggest
```
May I delete this line? `playerNum` was deprecated: https://github.com/smogon/pokemon-showdown/search?q=playerNum

also I will open other PR about this (for other languages)

Comment (AnnikaCodes) [reply translations/japanese/core-commands.ts:91]: Yes, thanks.

**Comment by Dirain1700** [comment]
> It seems like a few strings in helptickets.ts and minor-activities.ts are empty in Japanese translation, like this:-
> 
> `"Resolved": "",`
> 
> Are these still being worked on?

minor-activities.ts:

oops, I forgot. thanks!

helptickets.ts:

I'm not a global staff and no ones Japanese in global staffs, so I think it's unnecessary.
I'll try it; thanks!

**Comment by WeWuzNidokangz** [comment]
Oh, I see. But I think this one in helptickets.ts is displayed to users:-

`"Make sure you have provided the correct username, and if its meaning or why it is offensive is not obvious, please explain why it should not be allowed.": "",`

**Comment by AnnikaCodes** [comment]
Thanks!

**Pull request merged**